include/ddc/kernels/splines/spline_builder.hpp

```diff
@@ -142,6 +161,13 @@ class SplineBuilder
     int compute_offset(interpolation_domain_type const& interpolation_domain);
 
 public:
+    /**
+     * @brief Build a SplineBuilder acting on batched_interpolation_domain.
+     * 
+     * @param batched_interpolation_domain The domain on which are defined the interpolation points.
+     * @param cols_per_chunk An hyperparameter used by the slicer (internal to the solver) to define the size of a chunk of right-and-sides of the linear problem to be computed in parallel.
```

Comment:
I think it should be documented in MatrixSparse, not there. This is really solver-related, not splines. You agree ?

Reply:
> I think it should be documented in MatrixSparse

What is "it"? The slicer? I still don't know what it is so I don't think I can give an opinion :sweat_laugh:

> The default value is defined in MatrixSparse so I think we do not want to document it there in case we make a change.

I agree. The sentence here should be something like:
> This value is optional. If no value is provided then the default value is chosen by the requested solver.

I agree that we don't want to document the value itself twice in case we make changes. The important thing is that this message explains how to find the information

Reply:
Yeah slicer is the for loop inside MatrixSparse which slices the m-rhs in chunks of rhs to pass them one-after-the-other to ginkgo

Reply:
Ok. Please add a sentence to explain this

Reply:
I've added `(chunks are treated by the linear solver one-after-the-other).`, doing more looks redundant to me